<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1078</width>
    <height>750</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">.QWidget{
	 background-color:rgb(84 ,84 , 84);
}</string>
   </property>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>80</y>
      <width>631</width>
      <height>461</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="2" column="0">
      <widget class="QLabel" name="label_show_A1">
       <property name="frameShape">
        <enum>QFrame::Box</enum>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>290</y>
      <width>321</width>
      <height>401</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <widget class="QScrollArea" name="scrollArea">
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>200</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QScrollArea{\n background-color:rgb(84 ,84 , 84);}</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Sunken</enum>
       </property>
       <property name="widgetResizable">
        <bool>true</bool>
       </property>
       <widget class="QWidget" name="scrollAreaWidgetContents">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>-110</y>
          <width>302</width>
          <height>509</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>1</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>509</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QWidget
{
 background-color:rgb(84 ,84 , 84);
}</string>
        </property>
        <widget class="QLabel" name="label_5">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>301</width>
           <height>41</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">QLabel {
 background-color:rgb(48 ,48 , 48);
 color:rgb(255 ,255 ,255);
}</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="text">
          <string>Geometric Transformation</string>
         </property>
        </widget>
        <widget class="QPushButton" name="pb_trapezoidal">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>50</y>
           <width>281</width>
           <height>31</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton{
color:rgb(255,255,255);
}</string>
         </property>
         <property name="text">
          <string>Trapezoidal Transformation</string>
         </property>
        </widget>
        <widget class="QPushButton" name="pb_Circular">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>240</y>
           <width>281</width>
           <height>31</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton{
color:rgb(255,255,255);
}</string>
         </property>
         <property name="text">
          <string>Circular Transformation</string>
         </property>
        </widget>
        <widget class="QPushButton" name="pb_Wavelet">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>330</y>
           <width>281</width>
           <height>31</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton{
color:rgb(255,255,255);
}</string>
         </property>
         <property name="text">
          <string>Wavelet transform</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_11">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>280</y>
           <width>301</width>
           <height>41</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">QLabel {
 background-color:rgb(48 ,48 , 48);
 color:rgb(255 ,255 ,255);
}</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="text">
          <string> Image Fusion Using Wavelet Transform</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_20">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>380</y>
           <width>301</width>
           <height>41</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">QLabel {
 background-color:rgb(48 ,48 , 48);
 color:rgb(255 ,255 ,255);
}</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="text">
          <string>Hough Transform</string>
         </property>
        </widget>
        <widget class="QPushButton" name="pb_applykmeans">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>430</y>
           <width>281</width>
           <height>31</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton{
color:rgb(255,255,255);
}</string>
         </property>
         <property name="text">
          <string> Hough transform</string>
         </property>
        </widget>
        <widget class="QPushButton" name="pb_wavy">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>190</y>
           <width>281</width>
           <height>31</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton{
color:rgb(255,255,255);
}</string>
         </property>
         <property name="text">
          <string>Wavy Transformation</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_22">
         <property name="geometry">
          <rect>
           <x>70</x>
           <y>90</y>
           <width>171</width>
           <height>20</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">QLabel {
 color:rgb(255 ,255 ,255);
}</string>
         </property>
         <property name="text">
          <string>amplitude</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
        <widget class="QSlider" name="horizontalSlider_A">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>110</y>
           <width>281</width>
           <height>18</height>
          </rect>
         </property>
         <property name="minimum">
          <number>2</number>
         </property>
         <property name="maximum">
          <number>100</number>
         </property>
         <property name="singleStep">
          <number>2</number>
         </property>
         <property name="pageStep">
          <number>2</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
        <widget class="QSlider" name="horizontalSlider_F">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>150</y>
           <width>281</width>
           <height>18</height>
          </rect>
         </property>
         <property name="minimum">
          <number>30</number>
         </property>
         <property name="maximum">
          <number>300</number>
         </property>
         <property name="singleStep">
          <number>20</number>
         </property>
         <property name="pageStep">
          <number>20</number>
         </property>
         <property name="value">
          <number>30</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
        <widget class="QLabel" name="label_23">
         <property name="geometry">
          <rect>
           <x>70</x>
           <y>130</y>
           <width>171</width>
           <height>20</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">QLabel {
 color:rgb(255 ,255 ,255);
}</string>
         </property>
         <property name="text">
          <string>freqency</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label_origin">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>40</y>
      <width>321</width>
      <height>231</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_A">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>40</y>
      <width>631</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
 background-color:rgb(48 ,48 , 48);
 color:rgb(255 ,255 ,255);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_B">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>560</y>
      <width>631</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
 background-color:rgb(48 ,48 , 48);
 color:rgb(255 ,255 ,255);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_show_B1">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>600</y>
      <width>631</width>
      <height>81</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1078</width>
     <height>24</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QMenuBar
{
	 background-color:rgb(34 ,34 , 34);
color:rgb(255,255,255);
}</string>
   </property>
   <widget class="QMenu" name="file">
    <property name="styleSheet">
     <string notr="true">QMenu{

}</string>
    </property>
    <property name="title">
     <string>file</string>
    </property>
    <addaction name="action_import"/>
   </widget>
   <addaction name="file"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action_import">
   <property name="text">
    <string>import</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
